src/calculateRentalCost.js

```diff
@@ -28,6 +28,13 @@
  */
 
 function calculateRentalCost(days) {
+  if (days >= 1 && days < 3) {
+    return days * 40;
+  } else if (days >= 3 && days < 7) {
+    return days * 40 - 20;
+  } else if (days >= 7) {
+    return days * 40 - 50;
+  }
```

Comment:
Your solution works, but let's work on code a little bit.
I recommend you to move these numbers to variables with appropriate names. For example, number 50 could be stored in a variable with name sevenDaysDiscount.